src/lib/main/core/body/td-core.directive.ts

```diff
@@ -67,19 +68,12 @@ export class TdCoreDirective implements Td, OnInit, OnDestroy, OnChanges {
 				this.cd.detectChanges();
 				break;
 			}
-			case 'edit': {
-				this.element.classList.add(`${GRID_PREFIX}-edit`);
-
-				const link = this.cellService.build('body', this.column, 'edit');
-				link(this.viewContainerRef, this);
-				this.cd.markForCheck();
-				this.cd.detectChanges();
-				break;
-			}
+			case 'edit':
 			case 'change': {
-				this.element.classList.add(`${GRID_PREFIX}-change`);
-
-				const link = this.cellService.build('body', this.column, 'change');
+				const link = this.cellService.build('body', this.column, value);
+				if (link !== noop) {
```

Comment:
for edit we need to throw exception as it was in cellService, for change just do nothing